lib/api/accounts.js

```diff
@@ -113,15 +157,15 @@ module.exports = function (app) {
 				return cb(null, account);
 			}
 			return request.get({
-				url: `${app.get('lisk address')}/api/delegates/voters?publicKey=${account.publicKey}`,
+				url: `${app.get('lisk address')}/voters?publicKey=${account.publicKey}`,
 				json: true,
 			}, (err, response, body) => {
 				if (err || response.statusCode !== 200) {
 					return cb(err || 'Response was unsuccessful');
-				} else if (body.success === true) {
-					account.voters = (body.accounts !== undefined &&
-						body.accounts !== null && body.accounts.length > 0) ?
-						body.accounts : null;
+				} else if (body && Array.isArray(body.data)) {
+					account.voters = (body.data.voters !== undefined &&
```

Comment:
Simplify that check:
`account.voters = Array.isArray(body.data.voters) && body.data.voters.length > 0 ? body.data.voters : null;`